### DISCUSSION TITLE
[aptos token transfer] current support token transfer on aptos?

### DISCUSSION THREAD

**Discussion by weixuefeng** [trustwallet/wallet-core#3423] [Ideas & Feature request]
Current support token transfer on aptos? about https://aptos.dev/guides/nfts/mint-onchain-data/#transfer

**Comment by Milerius** [comment]
https://github.com/trustwallet/wallet-core/blob/573696ffde436485c6d8dabd7131c9a7a6baa775/tests/chains/Aptos/SignerTests.cpp#L19

**Comment by weixuefeng** [comment]
> https://aptos.dev/guides/nfts/mint-onchain-data/#transfer

FT transfer's  function is '0x3::token_transfers: transfer_with_opt_in', claimNft's function is  `0x3::token_transfers::claim_script`, i had not find 'transfer_with_opt_in' method.
